source/Halibut.Tests/SecureListenerFixture.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Net;
+using System.Threading;
+using FluentAssertions;
+using Halibut.Diagnostics;
+using Halibut.Transport;
+using NUnit.Framework;
+
+namespace Halibut.Tests
+{
+    public class SecureListenerFixture
+    {
+#if NETFRAMEWORK
```

Comment:
Why only `NETFRAMEWORK`? Pretty soon Halibut will be `netcoreapp` only.

Reply:
This is a weird one. Performance counters work in .net core 2.2 apps, but in .net core 2.2 unit test projects they always return 0. I'll see if I can get it working.